app/models/plant.rb

```diff
@@ -1,4 +1,6 @@
 class Plant < ApplicationRecord
+  validates_presence_of :name
```

Comment:
I actually prefer to use `validates :name, presence: true` because then you can add some other flags and not clutter the model. But for this purpose I have no issues with this. It works as expected.